ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master__nfd.yaml

```diff
@@ -21,13 +21,14 @@ resources:
       memory: 2Gi
 tests:
 - as: operator-e2e-master
+  cron: 0 23 * * *
```

Comment:
wrong place :) must be below the cluster_claim

Reply:
Fixed